src/lib/y2storage/encryption.rb

```diff
@@ -79,6 +81,23 @@ def in_etc?
       in_etc_crypttab?
     end
 
+    # Returns the absolute path to a file containing the encryption password
+    #
+    # To be used as a third field in the crypttab file
+    #
+    # @return [String]
+    def password_file
+      @password_file
+    end
+
+    # Sets the absolute path to a file containing the encryption password
+    #
+    # @param filename [String] absolute path to the file
```

Comment:
can I somehow reset it if I do not want to use file? ( just hint, I know it, but you should document it ).